lib/shoryuken/queue.rb

```diff
@@ -5,7 +5,11 @@ class Queue
     def initialize(client, name)
       self.name   = name
       self.client = client
-      self.url    = client.get_queue_url(queue_name: name).queue_url
+      begin
+        self.url = client.get_queue_url(queue_name: name).queue_url
+      rescue Aws::SQS::Errors::NonExistentQueue
+        raise StandardError.new "Non existing queue: #{name}. Please create it in advance in SQS."
```

Comment:
The default error makes no mention to the queue which is failing, which could be something unexpected if using ActiveJob or prefixing. This may not be the best approach but it's helpful to know where to start looking on Amazon when you get this error, hence the queue-name.